apps/dsp-app/src/styles/_elements.scss

```diff
@@ -342,6 +343,14 @@ a,
   // }
 }
 
+// file representation menu styling
+.representation-menu {
+  &.mdc-menu-surface {
+    @include menu-surface.fill-color(#292929);
```

Comment:
If possible, use the colors from the `styles/_config.scss` by importing it

Reply:
done in [c087480](https://github.com/dasch-swiss/dsp-das/pull/1118/commits/c087480f56c2e9497c217a8b2c51fffa3714a72a)